9/2581.py

```diff
@@ -0,0 +1,24 @@
+m = int(input())
+n = int(input())
+
+prime = []
+
+for i in range(m, n + 1):
+    check_prime = True
+    for j in range(2, i):
+        if i % j == 0:
+            check_prime = False
+            break
+    if i == 1:
+        check_prime = False
+    if check_prime == True:
+        prime.append(i)
+
+prime_sum = 0
+for i in range(len(prime)):
+    prime_sum += prime[i]
```

Comment:
이 경우에는 함수를 사용하는게 더 깔끔하겠군요?
```suggestion
    prime_sum = sum(prime)
```